src/index.html

```diff
@@ -1,14 +1,24 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+
+    <div class="container">
+
+      <div class="watch">
+
+        <div class="min-circle"></div>
+
+      </div>
+
+    </div>
+
+  </body>
```

Comment:
тобі не обов'язково робити відступи між блоками в яких не має великої кількості вкладених елементів. Візуально і так видно що вони вкладені
```suggestion
  <body>
    <div class="container">
      <div class="watch">
        <div class="min-circle"></div>
      </div>
    </div>
  </body>```

Reply:
поправил